specification/storage/resource-manager/Microsoft.Storage/stable/2021-08-01/storage.json

```diff
@@ -2586,6 +2839,18 @@
         "azureStorageSid": {
           "type": "string",
           "description": "Specifies the security identifier (SID) for Azure Storage."
+        },
+        "samAccountName": {
```

Comment:
Please update example for the 2 new properties of “ActiveDirectoryProperties”

Reply:
added in StorageAccountEnableAD.json